tests/baseline/cases/staged-simple-cloud.yml

```diff
@@ -0,0 +1,193 @@
+# A pushsource library testcase.
+#
+# This file was generated from a template.
+# To regenerate, run test_baseline.py with PUSHSOURCE_UPDATE_BASELINE=1.
+
+# URL of Source to test.
+url: "staged:{{ src_dir }}/tests/staged/data/simple_cloud"
+
+# Push items generated from above.
+items:
+- AmiPushItem:
+    access_endpoint_url: null
+    billing_codes: null
+    boot_mode: hybrid
+    build: null
+    build_info:
+      id: null
+      name: rhel-ec2
+      release: '1'
+      version: '9.4'
+    description: build2 sample
+    dest: []
```

Comment:
I don't think these `dest` should be empty. I know in the case of your client you're going to ignore the values anyway and overwrite them, which is fine, but I think the behavior in pushsource library should still be consistent for all content types produced by "staged" source.

i.e., can you please have the implementation fill in this value from the path, the same way it works elsewhere? (Meaning the value here should be `["starmap"]` .)